server/hotrod/pom.xml

```diff
@@ -19,10 +19,15 @@
    </properties>
 
    <dependencies>
-       <dependency>
+      <dependency>
          <groupId>org.infinispan</groupId>
          <artifactId>infinispan-server-core</artifactId>
       </dependency>
+      <dependency>
+         <groupId>org.infinispan</groupId>
+         <artifactId>infinispan-protocol-parser-generator-maven-plugin</artifactId>
+         <scope>provided</scope>
```

Comment:
I don't think this dependency is needed

Reply:
I think I need it when I run `mvn install -pl :infinispan-server-hotrod -am` to get it built even w/o having the plugin in local repo.

Reply:
The dependency in the plugins section should be enough, in fact this worked for me the first time after adding the phase: `mvn clean && mvn install -am -pl integrationtests/as-lucene-directory`